README.md

```diff
@@ -13,29 +13,29 @@ The [Teamscale](https://teamscale.com) pre-commit command line interface allows
 
 2. Install `teamscale-cli` via pip:
 
-```bash
-$ pip install teamscale-cli
-```
+    ```bash
+    $ pip install teamscale-cli
+    ```
 
-The `teamscale-cli` executable will be installed to `~/.local/bin/teamscale-cli`. Please ensure it is on your `PATH`.
+    The `teamscale-cli` executable will be installed to `~/.local/bin/teamscale-cli`. Please ensure it is on your `PATH`.
 
 3. Copy the example configuration file [.teamscale-precommit.config](./config/.teamscale-precommit.config) to the root directory of the repository you want to analyze. Edit it and insert all the necessary data. You can find your personal access token by opening Teamscale and clicking on your Avatar in the top right corner.
 
-4. Use `teamscale-cli` as a compile or build command in your editor. See below for instructions for popular editors. Provide a file or folder within your repository as input. The general invocation looks like this:
+5. The behavior of the client can be tweaked with several arguments. Run the client with the `-h` argument to get the usage.
```

Comment:
I think something went wrong there. It'd make more sense if we keep 4. and move 5. below the bash code snippet.

Reply:
-h is already mentioned it the end of the next section. I restored 4. as suggested :-)